### ISSUE TITLE
Feature Request: Create Docker Image for "production" use

### ISSUE THREAD

**Issue by Lapotor** [nabeelio/phpvms#1815]
### Summary
This feature request proposes the creation and provision of an Docker image for phpvms. The introduction of an Docker image will streamline the setup process for users, offering a more convenient and standardized environment for running the software, alongside the existing development setup facilitated by Laravel Sail.

### Context
Currently, phpvms supports two primary methods of setup:

1. Development environment using Laravel Sail.
2. Bare-metal setup as described in the project documentation.

While these methods are functional, the absence of an Docker image presents a barrier for users who prefer or require containerized environments for deployment and management. An Docker image would bridge this gap, enhancing the usability and adoption of phpvms.

### Benefits

- **Ease of Setup**: Users can quickly set up a phpvms environment without dealing with complex dependencies and configurations.
- **Consistency**: The Docker image ensures a consistent environment across different machines and setups, reducing the "it works on my machine" problem.
- **Portability**: Containerized environments can be easily moved across different systems, facilitating testing, development, and deployment.

### Conclusion
Providing an Docker image for phpvms will significantly enhance the user experience by simplifying the setup process and providing a reliable, consistent, and portable environment for managing virtual airlines. This addition aligns with modern development and deployment practices, ensuring phpvms remains accessible and easy to use for a broader audience.

**Comment by nabeelio** [comment]
Yeah, a production composer and Dockerfile is needed. I can work on this sometime based on a couple that I use

**Comment by arthurpar06** [comment]
Yes, that would definitely be a good idea. We might also consider FrankenPHP (https://frankenphp.dev/), which is based on Caddy and acts as a replacement for PHP-FPM. The Laravel integration seems quite simple (don't focus on the part with Laravel Octane).

**Comment by Lapotor** [comment]
Why not just use the official `php-fpm` or `php-apache` as base?
